xml/System/IntPtr.xml

```diff
@@ -244,23 +244,25 @@
+    <summary>Represents a signed integer where the bit-width is the same as a pointer.</summary>
     <remarks>
       <format type="text/markdown"><![CDATA[
 
 ## Remarks
 
- The <xref:System.IntPtr> type is designed to be an integer whose size is platform-specific. That is, an instance of this type is expected to be 32-bits on 32-bit hardware and operating systems, and 64-bits on 64-bit hardware and operating systems.
+ The <xref:System.IntPtr> type is designed to be an integer whose size is the same as a pointer. That is, an instance of this type is expected to be 32-bits in a 32-bit process and 64-bits in a 64-bit process.
 
  The <xref:System.IntPtr> type can be used by languages that support pointers and as a common means of referring to data between languages that do and do not support pointers.
 
  <xref:System.IntPtr> objects can also be used to hold handles. For example, instances of <xref:System.IntPtr> are used extensively in the <xref:System.IO.FileStream?displayProperty=nameWithType> class to hold file handles.
 
- The <xref:System.IntPtr> type is CLS-compliant, while the <xref:System.UIntPtr> type is not. Only the <xref:System.IntPtr> type is used in the common language runtime. The <xref:System.UIntPtr> type is provided mostly to maintain architectural symmetry with the <xref:System.IntPtr> type.
+ However, <xref:System.IntPtr> is not itself a pointer or a handle. It is simply an integer type that can be used as an interchange format for pointers and handles due to being the same size. Outside of specific interchange requirements, such as for passing data to a language that doesn't support pointers, a correctly typed pointer should be used to represent pointers and <xref:System.Runtime.InteropServices.SafeHandle> should be used to represent handles.
 
- This type implements the <xref:System.Runtime.Serialization.ISerializable>, and, in .NET 5 and later versions, the <xref:System.IFormattable> interfaces.
+ This type implements the <xref:System.Runtime.Serialization.ISerializable>. In .NET 5 and later versions, this type also implements the <xref:System.IFormattable> interfaces. In .NET 7 and later versions, this type also implements the <xref:System.Numerics.IBinaryInteger>, <xref:System.Numerics.IMinMaxValue>, and <xref:System.Numerics.ISignedNumber> interfaces.
 
  In C# starting from version 9.0, you can use the built-in `nint` type to define native-sized integers. This type is represented by the <xref:System.IntPtr> type internally and provides operations and conversions that are appropriate for integer types. For more information, see [nint and nuint types](/dotnet/csharp/language-reference/builtin-types/nint-nuint).
+ 
+ In C# starting from version 11.0 and when targeting the .NET 7 or later runtime, <xref:System.IntPtr> and `nint` are fully interchangeable with eachother in the same way that <xref:System.Int32> and `int` are interchangeable.
```

Comment:
I'm not sure if there is a better way to describe this change in relationship.

Reply:
It's clear, especially with the example of `int` given.

Reply:
It's fine, but you could also say "nint is an alias for System.IntPtr", because this seems to be common term for decribing this: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/integral-numeric-types

>In all of the table rows except the last two, each C# type keyword from the leftmost column is an alias for the corresponding .NET type